src/tiledb/cloud/sql/tiledb_cursor.py

```diff
@@ -0,0 +1,117 @@
+import pyarrow as pa
+
+from tiledb.cloud.sql._execution import exec
+from tiledb.cloud.sql.db_api_exceptions import DataError
+from tiledb.cloud.sql.db_api_exceptions import NotSupportedError
+from tiledb.cloud.sql.db_api_exceptions import ProgrammingError
+
+
+def _getDBType(dtype):
+    if pa.types.is_timestamp(dtype):
+        return "DATETIME"
+    elif pa.types.is_boolean(dtype):
+        return "BOOLEAN"
+    elif pa.types.is_string(dtype):
+        return "STRING"
+    elif pa.types.is_integer(dtype) or pa.types.is_floating(dtype):
+        return "NUMBER"
+    return "UNKNOWN"
+
+
+class Cursor:
+    def __init__(self):
+        self._results = None
+        self._row_index = 0
+        self.arraysize = 1
+        self._description_cache = None
+
+    def executemany(self, query):
+        self.execute(query)
+
+    def execute(self, query):
```

Comment:
This needs to accept a `parameters` optional parameter. The default value should probably be `()` (i.e., the empty tuple).

Likewise, `executemany` takes a sequence of `parameters`es, and it might look something like:

```python
def executemany(self, query, seq_of_parameters):
  for params in seq_of_parameters:
    self.execute(query, params)